### PR TITLE
Dep updates

### PR DESCRIPTION
**Pull request by jeff-mccoy**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

**Review by Racer159 — APPROVED** [review]
lgtm (if ci passes)

File: .github/workflows/test-ui.yml [hunk 1]
```diff
@@ -42,6 +42,9 @@ jobs:
           path: "~/Library/Caches/ms-playwright/"
```
Comment (Noxsios) [inline .github/workflows/test-ui.yml:42]: Is this path correct? https://github.com/defenseunicorns/zarf/actions/runs/4040760238/jobs/6946708240#step:8:19

File: docs-website/package.json [hunk 2]
```diff
@@ -49,5 +37,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "engines": {
+    "node": ">=16.14"
```
Comment (Noxsios) [inline docs-website/package.json:42]: If we are using Node 18 in CI, should we specify Node 18+ here as well?

Comment (jeff-mccoy) [reply docs-website/package.json:42]: removed

File: .github/workflows/test-ui.yml (hunk 1 above)
Comment (jeff-mccoy) [reply .github/workflows/test-ui.yml:42]: updated

**Review by Racer159 — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -70,7 +71,7 @@ jobs:
 
       # Set up AWS credentials for GoReleaser to upload backups of artifacts to S3
       - name: Set AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```
[the remark this answers is not in the document]
Comment (jeff-mccoy) [reply .github/workflows/release.yml:74]: That's what their docs offer--the default is actually node 12, but they added 16 in response to a gh deprecation warning.

https://github.com/aws-actions/configure-aws-credentials

[the remark this answers is not in the document]
Comment (jeff-mccoy) [reply .github/workflows/release.yml:74]: It has nothing to do with our use of node, it's what that action is written in as actions can be written in js.

**Review by Racer159 — APPROVED** [review]
lgtm

**Pull request merged**